examples/fire_alarm_demo.py

```diff
@@ -38,7 +39,8 @@ async def async_demo1(self) -> None:
                     print(
                         f"{key} ({item.get('name')}): status={item['device_state']} data={item['device_status_data']}"
                     )
-            except Exception:  # pylint: disable=broad-except
+            except Exception as e:  # pylint: disable=broad-except
+                print(f"Exception caught {e.args}")
                 pass
```

Comment:
Please rename `e` to err and remove the `pass` statement

Reply:
Done